tests/EventTest.php

```diff
@@ -165,6 +165,38 @@ public function testToArrayWithBreadcrumbs(): void
         $this->assertSame($breadcrumbs, $data['breadcrumbs']['values']);
     }
 
+    public function testToArrayWithMessageWithFormatted(): void
+    {
+        $expected = [
+            'message' => 'foo @bar',
+            'params' => ['@bar' => 'bar'],
+            'formatted' => 'foo bar',
+        ];
+
+        $event = new Event();
+        $event->setMessage('foo @bar', ['@bar' => 'bar'], 'foo bar');
+
+        $data = $event->toArray();
+
+        $this->assertArrayHasKey('message', $data);
+        $this->assertSame($expected, $data['message']);
+    }
+
+    public function testGetMessage(): void
```

Comment:
Instead of having multiple test conditions into a single test case please use a data provider